lib/qbwc/request/inventorywithsites.rb

```diff
@@ -12,6 +12,14 @@ def polling_current_items_xml(params, config)
           timestamp = params['quickbooks_since']
           session_id = Persistence::Session.save(config, 'polling' => timestamp)
 
+          if site_name(params) == ''
```

Comment:
what if we just omit site_name?

Reply:
Hmm, it would probably go to the else branch and try to just look for site names with an empty string. Nice catch! I'll add a check to see if the `quickbooks_site` is there at all